src/java.base/share/classes/java/lang/String.java

```diff
@@ -146,7 +161,7 @@
      * {@link Stable} is safe here, because value is never null.
      */
     @Stable
-    private final byte[] value;
+    private final byte[] @SideEffectFree value;
```

Comment:
@mernst 
This is an error that is introduced by `insert-annotations-to-source`. I will create a issue in AUF repository and l will correct this manually for the time-being.
I am able to reproduce this error in insolated `insert-annotations-to-source String.jaif String.java`